composer.json

```diff
@@ -20,8 +20,8 @@
         "php": "^7.4 || ^8.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^5",
```

Comment:
I was not able to run the phpstan `TypeInferenceTestCase` with the simple bridge.

is the bridge vital for this repo, or can we use plain phpunit?